pkg/sql/schemachanger/scplan/internal/opgen/opgen_namespace.go

```diff
@@ -22,6 +22,12 @@ func init() {
 		toPublic(
 			scpb.Status_ABSENT,
 			to(scpb.Status_PUBLIC,
+				emit(func(this *scpb.Namespace) *scop.SetDescriptorName {
+					return &scop.SetDescriptorName{
```

Comment:
suggestion: rename this op to `SetNameInDescriptor` to differentiate it from `AddDescriptorName`.